configuration/globalproperties/gp_zl.xml

```diff
@@ -48,5 +48,9 @@
             <property>labworkflowowa.labResultsEncounterTypes</property>
             <value>${encounterType.labSpecimenCollection.uuid},${encounterType.laboratoryResults.uuid},${encounterType.covidIntake.uuid},${encounterType.covidFollowup.uuid},${encounterType.zlHivIntake.uuid},${encounterType.zlHivFollowup.uuid}</value>
         </globalProperty>
+        <globalProperty>
+            <property>registrationapp.paperRecordIdentifierDefinition</property>
+            <value>paperrecord.patientDataDefinition.paperRecordIdentifier</value>
+        </globalProperty>
```

Comment:
Following on with comments on the PR in Registration app, this should be the default value, so we shouldn't need this here... 

I'd suggest instead, in gp.xml in config-pihemr, set this value to the null/empty and then here you should only need the two lines before in pih-config-haiti and pih-config-mirebalais where you set it to "paperrecord.patientDataDefinition.paperRecordIdentifier"